### DISCUSSION TITLE
Connections list on bitfocus.io doesn't seem to support the v3 help layout.

### DISCUSSION THREAD

**Discussion by jwetzell** [bitfocus/companion#2665] [Q&A]
Not sure where else to put this but companion modules that are using the v3 layout for files don't have their HELP.md properly loaded in the [connections list](https://bitfocus.io/connections) on bitfocus.io. I understand that the HELP.md moved in the v3 format to underneath a companion folder in the module. The site just does not seem to accommodate that.

**Comment by peternewman** [comment]
Previously raised as an issue by me here:
https://github.com/bitfocus/companion/issues/2534

Probably an easy enough fix, but the code isn't explicitly open source anywhere (I've not looked to see if it's minified or not).

**Reply by jwetzell** [reply]
Dang I even tried looking for an existing issue (wasn't quite sure what to search). Definitely minified, but definitely should be an easy fix.

> <img width="838" alt="Screen Shot 2023-11-27 at 8 08 06 PM" src="https://github.com/bitfocus/companion/assets/18341515/5b42f1ef-56be-4d0d-a79f-085f810d88a9">

**Comment by Julusian** [comment]
This is fixed now.

It is partially powered by https://github.com/bitfocus/companion-worker-modules, partially some non public code

**Reply by peternewman** [reply]
I assume the juicy bit fetching the actual READMEs isn't in there from the looks of things @Julusian, just the bit to get the list of modules?

Possibly related, or maybe it should be a new issue/discussion, but I spotted that e.g. https://bitfocus.io/connections/agf-characterworks doesn't appear in the main connection list?

Would that be somehow down to this:
https://github.com/bitfocus/companion-module-agf-characterworks/pull/7

As a few other modules seem to also have incorrect data and not show in the listing...
https://github.com/search?q=org%3Abitfocus%20git%2Bhttps%3A%2F%2Fgithub.com%2Fbitfocus%2Fcompanion-module-your-module-name.git&type=code

**Reply by Julusian** [reply]
Yeah, the code looks like it is filtering out any modules with `companion-module-your-module-name` in their manifest data

**Reply by Julusian** [reply]
Going forward, the module inclusion workflow will fail it a module has that string in its manifest.json, so this problem should go away once the existing occurences are fixed
